SpatialGDK/Source/SpatialGDK/Private/Interop/SkeletonEntities.cpp

```diff
@@ -0,0 +1,501 @@
+			}
+
+			const Worker_EntityId CreatedSkeletonEntityId = Kvp.Key;
+			AActor& SkeletonEntityActor = *Kvp.Value;
+
+			const VirtualWorkerId AuthWorkerId = NetDriver->LoadBalanceStrategy->WhoShouldHaveAuthority(SkeletonEntityActor);
+
+			const Worker_PartitionId AuthWorkerPartition =
+				NetDriver->VirtualWorkerTranslator->GetPartitionEntityForVirtualWorker(AuthWorkerId);
+
+			UE_LOG(LogSpatialSkeletonEntityCreator, Log,
+				   TEXT("EntityId: %lld Delegating skeleton entity to VirtualWorker %d Partition %lld"), CreatedSkeletonEntityId,
+				   AuthWorkerId, AuthWorkerPartition);
+
+			AuthorityDelegation AuthorityDelegationComponent;
+			AuthorityDelegationComponent.Delegations.Emplace(SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID, AuthWorkerPartition);
+
+			EntitiesToPopulatingWorkers.Emplace(CreatedSkeletonEntityId, AuthWorkerId);
+
+			NetDriver->Connection->GetCoordinator().SendComponentUpdate(
+				CreatedSkeletonEntityId,
+				ComponentUpdate(OwningComponentUpdatePtr(AuthorityDelegationComponent.CreateAuthorityDelegationUpdate().schema_type),
+								AuthorityDelegation::ComponentId),
+				{});
+		}
+
+		SkeletonEntitiesToDelegate.Empty();
+
+		UE_LOG(LogSpatialSkeletonEntityCreator, Log, TEXT("Finished delegating entities, moving on to manifest"));
+
+		Stage = EStage::SigningManifest;
+	}
+
+	if (Stage == EStage::SigningManifest)
+	{
+		TMap<VirtualWorkerId, TArray<Worker_EntityId>> WorkerToEntitiesToPopulate;
+		// Every worker should get a manifest to proceed through the skeleton entity flow.
+		for (const auto& VirtualWorker : NetDriver->WellKnownEntitySystem->VirtualWorkerTranslationManager->GetVirtualWorkerMapping())
+		{
+			WorkerToEntitiesToPopulate.FindOrAdd(VirtualWorker.Key);
+		}
+		for (const auto& SkeletonEntityToPopulatingWorker : EntitiesToPopulatingWorkers)
+		{
+			WorkerToEntitiesToPopulate.FindOrAdd(SkeletonEntityToPopulatingWorker.Value).Add(SkeletonEntityToPopulatingWorker.Key);
+		}
+		for (const auto& WorkerToEntitiesToPopulateEl : WorkerToEntitiesToPopulate)
+		{
+			const VirtualWorkerId WorkerId = WorkerToEntitiesToPopulateEl.Key;
+			const TArray<Worker_EntityId>& EntitiesToPopulate = WorkerToEntitiesToPopulateEl.Value;
+
+			TArray<ComponentData> SkeletonEntityManifestComponents;
+
+			{
+				FSkeletonEntityManifest Manifest;
+				Manifest.EntitiesToPopulate = EntitiesToPopulate;
+
+				SkeletonEntityManifestComponents.Emplace(Manifest.CreateComponentData());
+			}
+
+			const Worker_PartitionId ServerWorkerPartitionId =
+				NetDriver->VirtualWorkerTranslator->GetPartitionEntityForVirtualWorker(WorkerId);
+
+			{
+				const Worker_PartitionId ThisWorkerPartitionId = NetDriver->VirtualWorkerTranslator->GetClaimedPartitionId();
+				AuthorityDelegation Authority;
+				Authority.Delegations.Emplace(SpatialConstants::SKELETON_ENTITY_MANIFEST_AUTH_COMPONENT_SET_ID, ServerWorkerPartitionId);
+				Authority.Delegations.Emplace(SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID, ThisWorkerPartitionId);
```

Comment:
SPATIALOS_WELLKNOWN_COMPONENTSET_ID instead? You may not need the full server auth set.

Reply:
Yup, this one should work - this is the manifest so we don't need any of the actor components/tags. Thanks!